smoketest/tests/register_token.rs

```diff
@@ -1,27 +1,27 @@
 use codec::Encode;
 use ethers::{
-	core::types::Address,
-	middleware::SignerMiddleware,
-	providers::{Http, Provider},
-	signers::{LocalWallet, Signer},
-	utils::parse_units,
-	utils::rlp::Encodable,
+    core::types::Address,
+    middleware::SignerMiddleware,
+    providers::{Http, Provider},
+    signers::{LocalWallet, Signer},
+    utils::parse_units,
+    utils::rlp::Encodable,
 };
 use futures::StreamExt;
 use hex_literal::hex;
 use snowbridge_smoketest::{
-	contracts::{i_gateway, weth9},
-	parachains::assethub::api::{
-		foreign_assets::events::Created,
-		runtime_types::xcm::v3::{
-			junction::{
-				Junction::{AccountKey20, GlobalConsensus},
-				NetworkId,
-			},
-			junctions::Junctions::X3,
-			multilocation::MultiLocation,
-		},
-	},
+    contracts::{i_gateway, weth9},
```

Comment:
We tabs instead of spaces for indent.

I think we need to copy the `rustfmt.toml` from the parachains dir to this smoketests crate.

Reply:
Good point! https://github.com/Snowfork/snowbridge/pull/972/commits/cc9b1fc2aa6f50764287be80db7eb4e8666d4fe7

Reply:
One single  `rustfmt.toml` in root dir(snowbridge) should applies to all sub modules. And just checked our rules are exactly the same as Parity's(which in polkadot-sdk).

Reply:
What do you run to format it then? If I run 

`chronic cargo +$RUST_NIGHTLY_VERSION fmt`

from the snowbridge dir I get ``cargo metadata` exited with an error: error: could not find `Cargo.toml` in `snowbridge` or any parent directory`

Reply:
Just run in parachain dir, it will respect the config(rustfmt.toml or .rustfmt.toml) in any other parent directory. Or place the rule file in [global config directory](https://docs.rs/dirs/4.0.0/dirs/fn.config_dir.html) (e.g. .config/rustfmt/) will also do the job.